cluster/gce/gci/configure.sh

```diff
@@ -490,7 +490,10 @@ function install-containerd-ubuntu {
   # Override to latest versions of containerd and runc
   systemctl stop containerd
   if [[ ! -z "${UBUNTU_INSTALL_CONTAINERD_VERSION:-}" ]]; then
-    curl -fsSL "https://github.com/containerd/containerd/releases/download/${UBUNTU_INSTALL_CONTAINERD_VERSION}/containerd-${UBUNTU_INSTALL_CONTAINERD_VERSION:1}.linux-amd64.tar.gz" | tar --overwrite -xzv -C /usr/
+    # containerd versions have slightly different url(s), so try both
+    ( curl -fsSL "https://github.com/containerd/containerd/releases/download/${UBUNTU_INSTALL_CONTAINERD_VERSION}/containerd-${UBUNTU_INSTALL_CONTAINERD_VERSION:1}-linux-amd64.tar.gz" || \
```

Comment:
too hacky :) ??
```
curl -fsSL "https://github.com/containerd/containerd/releases/download/{v,}${UBUNTU_INSTALL_CONTAINERD_VERSION}/containerd-${UBUNTU_INSTALL_CONTAINERD_VERSION}{-,.}linux-amd64.tar.gz"
```
https://unix.stackexchange.com/a/243141

Reply:
not sure if this will work as if one of the urls fail, the curl process returns a bad exit code which will cause the job to fail. in other words, this will work only if both files can be downloaded i think